compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtInterfaces.td

```diff
@@ -40,4 +41,118 @@ def LinalgExtInterface : OpInterface<"LinalgExtOp"> {
   let verify = [{ return detail::verifyLinalgExtOpInterface($_op); }];
 }
 
+// Interface that allows for fusion of both LinalgExt and Linalg ops.
+def LinalgFusionInterface : OpInterface<"LinalgFusionOpInterface", [DestinationStyleOpInterface]> {
+  let methods = [
+    //===------------------------------------------------------------------===//
+    // Interface methods for fusion. 
+    //===------------------------------------------------------------------===//
+    InterfaceMethod<
+      /*desc=*/[{
+        Return an AffineMap for each operand or `std::nullopt` if the operand
+        does not have an indexing map representation.
+      }],
+      /*retTy=*/"SmallVector<std::optional<AffineMap>>",
```

Comment:
Just return a `SmallVector<AffineMap>`. The `AffineMap` can be `nullptr` to indicate not having an indexing map representation.